pkgs/tools/security/spectre-cli/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, gcc11Stdenv
+, fetchFromGitLab
+, cmake
+, libsodium
+, json_c
+, ncurses
+, libxml2
+, jq
+, ...
+}:
+
+gcc11Stdenv.mkDerivation {
+  pname = "spectre-cli";
+  version = "unstable-2021-02-05";
+
+  src = fetchFromGitLab {
+    owner = "spectre.app";
+    repo = "cli";
+    rev = "a5e7aab28f44b90e5bd1204126339a81f64942d2";
+    sha256 = "1kda4jj7sh940fpy9cqfgzmwjlwnkr75lqgm3addpzpblnw6vvwr";
+    fetchSubmodules = true;
+    deepClone = true;
+    leaveDotGit = true;
```

```suggestion
    sha256 = "1hp4l1rhg7bzgx0hcai08rvcy6l9645sfngy2cr96l1bpypcld5i";
    fetchSubmodules = true;
```
Is there a particular reason the build needs the `.git` directory?

---

No reason, forgot to remove them. `fetchSubmodules` is still needed as you noticed :-)